source/application/AMDatamanAppController.cpp

```diff
@@ -356,16 +356,16 @@ bool AMDatamanAppController::startupOnFirstTime()
 			return false;
 	}
 	else{
-		AMFirstTimeWizard ftw;
+		AMFirstTimeWizard *firstTimeWizard = new AMFirstTimeWizard(AMUserSettings::userBasedDataStorage);
 
```

Comment:
Yes it will since it was changed to a pointer it will need to be cleaned up before leaving the code block.